.github/workflows/build-openapi.yaml

```diff
@@ -0,0 +1,37 @@
+name: Run .NET Project with Swagger
+
+on:
+  workflow_call:
+    inputs:
+      dry-run:
+        description: "An indication of whether to commit/publish results"
+        required: true
+        type: string
+      is-dependabot:
+        description: "An indication of a dependabot pull request"
+        required: true
+        type: string
+
+jobs:
+  run-dotnet-project:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Resolve image version
+        uses: Energinet-DataHub/acorn-actions/actions/docker-image-version@v1
+
+      - name: Resolve image name
+        shell: bash
+        run: echo "name=$(yq '.name' 'domains/configuration.yaml')" | tee -a $GITHUB_ENV
+
+      - name: Build and push Docker image
+        uses: Energinet-Datahub/acorn-actions/actions/docker-build-and-push@v1
+        with:
+          dockerfile: ./domains/Dockerfile
```

Comment:
We should aim to use the dockerfile provided by ratatosk. It has been been designed for our needs, while being more performant and secure than the default offerings you can find online.

Reply:
We only need to run dotnet run, no need to change their dockerfile to do it